# Generated for NaoQi method ALSentinel.run
# Monitors buttons and Battery.

---


